doc/oci.rst

```diff
@@ -0,0 +1,140 @@
+Component Content in an OCI Registry
+====================================
+
+Component descriptors and local content delivered with a component are stored
+in an OCI registry as OCI artefact. The component descriptor may describe
+refernces to to other artefacts in any repository what type is supported
+by an access type, regardless of the type of the artefact. There are dedicated
+access types to refer to artefacts stored in an OCI registry.
+
+Storing Components
+------------------
+
+The content of a component is stored in the OCI registry as OCI artefact of the media type ``application/vnd.gardener.cloud.cnudie.component.config.v1+json``. Hereby the content consists at least of the component descriptor itself, which is basically stored as blob described as first layer in the oci manifest.
+The media type is used as type for the config blob described in the oci manifest. The blob confains a config json describing the digest of the component descriptor blob (first layer).
+
+The component descriptor blob uses the media type ``application/vnd.gardener.cloud.cnudie.component-descriptor.v2+json``.
```

Comment:
shouldn't the blob media type be `application/vnd.gardener.cloud.cnudie.component-descriptor.v2+json+tar` as it is a tar not a json?

Reply:
no, it is no tar, the compdesc is used as it is as blob

Reply:
ok, should be a tar

Reply:
I thought that the blob is a tar where the first entry is a file called `component-descriptor.yaml` or has this changed?

Reply:
you are right